src/Tizen.Applications.Cion/Tizen.Applications/ClientBase.cs

```diff
@@ -268,7 +268,7 @@ public Task<PayloadAsyncResult> SendPayloadAsync(Payload payload)
             }
 
             TaskCompletionSource<PayloadAsyncResult> tcs = new TaskCompletionSource<PayloadAsyncResult>();
-            _tcsDictionary[payload.Id] = tcs;
+            _tcsDictionary[payload?.Id] = tcs;
```

Comment:
If payload is null,  this will be
_tcsDictionary[null] = tcs;
Is it right?

Reply:
I fixed it other way. please review again.